src/jobs/dto/update-jobstatus.dto.ts

```diff
@@ -3,14 +3,23 @@ import { IsOptional, IsString } from "class-validator";
 
 @ApiTags("jobs")
 export class UpdateJobStatusDto {
+  @ApiProperty({
+    type: String,
+    required: true,
+    description: "Updated job status message for the current status.",
+  })
+  @IsString()
+  readonly jobStatusMessage: string;
+
   @ApiProperty({
     type: String,
     required: true,
     description: "Updated job status code for the current status.",
   })
   @IsString()
-  readonly jobStatus: string;
+  readonly jobStatusCode: string;
 
+  // TBD are 'message' and 'token' needed?
```

Comment:
Message, yes. It's the human-readable interpretation of the code. Maybe it could be optional in the DTO (but including it should be best practice).

@nitrosx Are you using token? Shouldn't authorization be done in the `Bearer` header? (This can be a separate issue, not necessarily part of this PR)

Reply:
There is already `jobStatusMessage`, which is the human-readable interpretation of `jobStatusCode`. Why is `message` also needed?